README.md

```diff
@@ -52,61 +52,73 @@ Dooing comes with sensible defaults that you can override:
 
     -- Window settings
     window = {
-		width = 55,
-		height = 20,
-		border = "rounded",
-		padding = {
-			top = 1,
-			bottom = 1,
-			left = 2,
-			right = 2,
-		},
-	},
-
-    -- To-do formatting
-	formatting = {
```

Comment:
this configuraton can´t be removed, added in the latest update

Reply:
Oops, sorry for the mistake, I just re-added it!